src/components/admin/groups/group-list/GroupList.scss

```diff
@@ -0,0 +1,18 @@
+.group-list {
+  width: 100%;
+  max-width: 700px;
+  margin: 0 auto;
```

Comment:
maybe make this a mixin... not sure what this is, but I've seen a lot of :
"
width: 100%;
max-width: 700px;
margin: 0 auto;
"

Reply:
i.e in .exercise-form, &__success-message, .exercise-list, .word-list, etc.